docs/en/helm/component.md

```diff
@@ -11,7 +11,7 @@ so we provide a chart which only includes minimal dependencies KubeVela relies o
 
 Install the minimal flux2 chart provided by KubeVela:
 ```shell
-helm install --create-namespace -n flux-system helm-flux http://oam.dev/catalog/helm-flux2-0.1.0.tgz
+$ helm install --create-namespace -n flux-system helm-flux http://oam.dev/catalog/helm-flux2-0.1.0.tgz
```

Comment:
without the `$` , the user can copy-paste  directly

Reply:
Then we should have consistency, for now all other docs have $

Reply:
Some of the commands along with output should keep `$`